cmd/psa/verify_as_attester.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package psa
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/veraison/apiclient/verification"
+	"github.com/veraison/evcli/common"
+	"github.com/veraison/go-cose"
+	"github.com/veraison/psatoken"
+)
+
+var (
+	attesterClaimsFile *string
+	attesterKeyFile    *string
+	attesterAPIURL     *string
+	attesterNonceSz    *uint
+)
+
+var (
+	attesterVeraisonClient common.IVeraisonClient = &verification.ChallengeResponseConfig{}
+	attesterCmd                                   = NewAttesterCmd(common.Fs, attesterVeraisonClient)
+)
+
+func NewAttesterCmd(fs afero.Fs, attesterVeraisonClient common.IVeraisonClient) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "attester",
+		Short: "Emulate an Attester",
+		Long: `
+	This command implements the "attester mode" of interaction, where the
+	verifier is the protocol challenger.  Here, the nonce is provided by the API
```

Comment:
nonce is provided by the Veraison API Server

Reply:
ok, done